enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,302 @@
+- A non-x86 IPI installer would serve as a first step towards improving the ease of deployments on non-x86 hadware.
+- Power hardware is especially targeted to AI and secure workloads, so building out cloud capabilities will help expand
+our workload footprint.
+- IPI is generally easier to install and set up quickly for demos and proof of concepts, when can help lower the barrier
+of entry for power optimized workloads.
+
+### Goals
+
+- Provide a way to install OpenShift on Power VS infrastructure using the OpenShift installer with IPI provisioned
+against IBM Cloud APIs.
+- Implement the machine-api controllers to provide scaling and managing the cluster nodes post install.
+
+### Non-Goals
+
+- UPI support. We've already documented how to install on pp64le hardware using UPI.
+
+## Proposal
+
+In order to leverage the Power VS platform, we will make use of the Power VS support in the [IBM Cloud Terraform
+Provider][ibm-cloud-terraform-provider]. This provider will interface with IBM Cloud APIs and ultimately enable the
+OpenShift Installer to provision VM resources using Power VS, that will be used as worker and masters of the cluster.
+Unlike some of the existing providers, this depends on image availability of CoreOS in the Power VS cloud in the OVA
+format. There are currently 2 proposals of how that can be achieved.
+
+The expected workflow would be as follows:
+1. The installer is invoked with credentials for ones IBM Cloud account.
+2. Once invoked, the installer interacts with a terraform provider to create a new Power VS service.
+3. The installer or terraform provider will then import a CoreOS OVA boot image file into the Power VS service from IBM
+Cloud Object Storage, which creates a corresponding boot image.
+4. Once imported the installer can start building the private network for the nodes.
+5. The bootstrap and master nodes will be created on in the private network. These are called Power VS instances, and
+they get created using the CoreOS boot image.
+6. In order to expose this private network and provide load balancing, the IBM Cloud APIs will leverage the IBM Cloud
+[Direct Link][direct-link-website] offering. This will allow for both the ingress load balancer and the cluster nodes to
+talk over the private network on a high-speed link.
+7. Bootstrapping completes as normal.
+8. The workers will be added using the openshift/[cluster-\*machine-\*] controllers and operators based on the openshift
+MAPI model. A Power VS controller needs to be implemented and hooked into the [OpenShift Machine
+Operator][openshift-machine-operator].
+9. The cluster teardown can make use of the existing [go client for Power][power-go-client] via `openshift-install
+destroy cluster`.
+
+### User Stories
+
+- A user of this installer would like to see a cluster running within an hour of kicking off the installer.
+- The expectation is that this is fully automated.
+
+### Implementation Details/Notes/Constraints
+
```

Comment:
@clnperez @mkumatag could we also add the high level diagram that was in the presentation explaining the high level data flow and the interaction of different components in the IBMCloud?

Reply:
Is this covered by the image under "Design Details"?

Reply:
i was referring to the other diagram you had on your slides (slide 6) which explains the overall interaction of the IBMCloud specific components and the Power VS specific components.

Reply:
Awaiting ack to add this